platforms/access-bridge/src/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { IncomingMessage } from 'node:http';
+
+import { BadRequestError } from './errors.js';
+
+export function isValidSiteId(siteId: string): boolean {
+  // Regular expression to match exactly 8 alphanumeric characters
+  const alphanumericRegex = /^[a-zA-Z0-9]{8}$/;
+  return alphanumericRegex.test(siteId);
+}
+
+// parse request body provided in an API call
+export const parseJsonBody = <T>(req: IncomingMessage): Promise<T> => {
```

Comment:
If you use express framework or other libs, you can avoid this extra implementation logic